tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +49,9 @@ def test_removing_not_found_is_false():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
+    assert result == False
```

Comment:
Asserting that `result` is `False` works well here because that's specifically what's returned from the function 👍🏽 
You could also add a check that the vendor's inventory list is unchanged:
```python
    assert len(vendor.inventory) == 3
```